pom.xml

```diff
@@ -214,6 +215,16 @@
                 <artifactId>shiro-core</artifactId>
                 <version>${apache.shiro.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.commons</groupId>
+                <artifactId>commons-lang3</artifactId>
+                <version>${commons-lang3.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>commons-lang</groupId>
+                <artifactId>commons-lang</artifactId>
+                <version>${commons-lang.version}</version>
```

Comment:
We may need a tech debt ticket to clean up commons-lang dependencies with this change